libdevcore/Log.h

```diff
@@ -26,7 +26,6 @@
 #include <ctime>
 #include <chrono>
 #include <boost/thread.hpp>
-#include <boost/asio.hpp>
```

Comment:
including all of boost::asio like this pulls an implicit link dependency on boost_regex